plugins/crossfade/front.js

```diff
@@ -8,13 +8,7 @@ let transitionAudio; // Howler audio used to fade out the current music
 let firstVideo = true;
 let waitForTransition;
 
-// Crossfade options that can be overridden in plugin options
-let crossfadeOptions = {
-	fadeInDuration: 1500, // ms
-	fadeOutDuration: 5000, // ms
-	exitMusicBeforeEnd: 10, // s
-	fadeScaling: "linear",
-};
+const defaultOptions = require('../../config/defaults').plugins.crossfade;
```

Comment:
nit: considering we override it with the plugin options, I would name it differently (to avoid the confusion about the defaults), eg
```suggestion
const pluginOptions = require('../../config/defaults').plugins.crossfade;
```

Reply:
sure but won't matter after https://github.com/th-ch/youtube-music/pull/1064